settings/acceptable_versions.go

```diff
@@ -0,0 +1,51 @@
+package settings
+
+import (
+	"fmt"
+	"github.com/packwiz/packwiz/core"
+	"github.com/spf13/cobra"
+	"golang.org/x/exp/slices"
+	"os"
+	"strings"
+)
+
+var acceptableVersionsCommand = &cobra.Command{
+	Use:     "acceptable-versions",
+	Short:   "Manage your pack's acceptable versions. This must be a comma seperated list of Minecraft versions, e.g. 1.16.5,1.16.4,1.16.3",
+	Aliases: []string{"av"},
+	Args:    cobra.ExactArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		modpack, err := core.LoadPack()
+		if err != nil {
+			// Check if it's a no sucj fiole or directory error
```

Comment:
```suggestion
			// Check if it's a no such file or directory error
```
I wonder if a helper function could be used to apply this for every command? Not sure how best to structure it, but it would be nice to check for this everywhere.

Reply:
Yeah, I agree. (and you can blame my uncaffinated brain for that typo). Probably would want a core.LoadPackOrInformUser-esque function.

Reply:
Actually, thinking about it... it might be a good idea to have commands return errors rather than calling `os.Exit` themselves, then having `cmd.Execute()` determine what the error is and print an appropriate message? This is quite a big change to do for existing commands so I wouldn't bother with it for now, but definitely something I will do in a future rewrite (particularly if I move to Rust, as there are some [really](https://lib.rs/crates/color-eyre) [nice](https://lib.rs/crates/miette) error formatting libraries).